pkg/e2/v1beta1/balancer/resolver.go

```diff
@@ -195,7 +195,8 @@ func (r *Resolver) updateState() {
 	log.Infof("New resolver addresses: %+v", addresses)
 
 	// Update the resolver state with list of E2T addresses annotated by nodes for which they are masters
-	r.clientConn.UpdateState(resolver.State{
+	// TODO - this call sometimes returns an error in the 1.41 version of grpc. Need to figure out why
+	_ = r.clientConn.UpdateState(resolver.State{
```

Comment:
what is the error?

Reply:
A generic "something was bad" error. This was Thomas' interim solution, the function did not return an error previously

Reply:
Event though the error is being returned, all indications are that the state was indeed updated and a new picker is created as a result.